Doesn't work with the new resolver

Compiling an empty project with the following `Cargo.toml` throws numerous errors.

```
[package]
name = "test"
version = "0.1.0"
edition = "2018"
resolver = "2"

[dependencies]
diesel = { version = "1.4.5", features = ["postgres"] }
diesel-derive-newtype = { version = "0.1.2" }
```

Errors look like
```
error[E0432]: unresolved import `diesel`
  --> /home/me/.cargo/registry/src/github.com-1ecc6299db9ec823/diesel-1.4.6/src/sql_types/mod.rs:37:48
   |
37 | #[derive(Debug, Clone, Copy, Default, QueryId, SqlType)]
   |                                                ^^^^^^^ could not find `pg` in `diesel`
   |
   = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
```
It works fine without `resolver = "2"`

I guess the previous resolver was implicitly activating the `postgres` feature of the diesel dependency of this crate.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -16,7 +16,6 @@ travis-ci = { repository = "quodlibetor/diesel-derive-newtype", branch = "master
 proc-macro2 = "0.4"
 syn = "0.14"
 quote = "0.6"
-diesel = "1"
 
 [dev-dependencies]
 diesel = { version = "1", features = ["sqlite"] }

